web/v1/redshift/sql-runner/sql/standard/02-page-views/01-main/00-setup.sql

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
(just on the filename,will #8 be included?)

Reply:
Good spot - @bill-warner I forgot this one actually, should be a quick one to include though if you can. :)